zerver/tests/test_subs.py

```diff
@@ -2927,7 +2927,7 @@ def test_query_count(self) -> None:
         self.assertEqual({dct["stream_id"] for dct in default_streams}, new_stream_ids)
 
         # Make sure our query isn't some bloated select_related query.
-        self.assertLess(len(queries[0].sql), 800)
+        self.assertLess(len(queries[0].sql), 825)
```

Comment:
It took me a couple readings to realize this is about the length of _one query, in characters_ -- not the _number of queries_.  This test came in during d6ef94f63f67e546bd8d7e19e589ab1e6dc91a43, where the gross query was caused by a bare `select_related()`.  7d0c12aa3acc90e5ab33439cf88efdb086d0c14b has enforced that we do not allow that construction anymore, so IMHO this weird limit test should be removed, rather than bumping it up every time we add something useful to the query.

Reply:
Removed.